admin/app/jobs/FastlyCloudwatchLoadJob.scala

```diff
@@ -10,21 +13,50 @@ object FastlyCloudwatchLoadJob extends ExecutionContexts with Logging {
   // Samples in CloudWatch are additive so we want to limit duplicate reporting.
   // We do not want to corrupt the past either, so set a default value (the most
   // recent 15 minutes of results are unstable).
+
+  // This key is (service, name, region)
```

Comment:
+1 for documentation, but if possible a case class would document it in the code instead of a comment